### ISSUE TITLE
Cache key validating not working

### ISSUE THREAD

**Issue by WildEgo** [supergrecko/RiotQuest#53]
**Describe the bug**
Cache key validating not working

**To Reproduce**
1. Called freshly account (Works correctly)
2. Called ranked stats (Works correctly I think)
3. Called empty match history (I think this is where it breaks)

**Expected behavior**
I shouldn't get the error message I get on the snippet below

**Desktop (please complete the following information):**
 - OS: Windows 10
 - PHP Version: 7.2.10
 - RiotQuest Version: 1.3.5

**Additional context**
Error snippet

```php
#0 E:\XAMPP\htdocs\******\vendor\symfony\cache\Traits\AbstractTrait.php(262): Symfony\Component\Cache\CacheItem::validateKey('riotquest.reque...')
#1 E:\XAMPP\htdocs\******\vendor\symfony\cache\Traits\AbstractAdapterTrait.php(44): Symfony\Component\Cache\Adapter\AbstractAdapter-&gt;getId('riotquest.reque...')
#2 E:\XAMPP\htdocs\******\vendor\symfony\cache-contracts\CacheTrait.php(48): Symfony\Component\Cache\Adapter\AbstractAdapter-&gt;getItem('riotquest.reque...')
#3 E:\XAMPP\htdocs\******\vendor\symfony\cache\Traits\ContractsTrait.php(95): Symfony\Component\Cache\Adapter\AbstractAdapter-&gt;doGet(Object(Symfony\Component\Cache\Adapter\FilesystemAdapter), 'riotquest.reque...', Object(Closure), 1, NULL, NULL)
#4 E:\XAMPP\htdocs\******\vendor\symfony\cache-contracts\CacheTrait.php(30): Symfony\Component\Cache\Adapter\AbstractAdapter-&gt;doGet(Object(Symfony\Component\Cache\Adapter\FilesystemAdapter), 'riotquest.reque...', Object(Closure), 1, NULL)
#5 E:\XAMPP\htdocs\******\vendor\supergrecko\riot-quest\src\RiotQuest\Components\Engine\Request.php(222): Symfony\Component\Cache\Adapter\AbstractAdapter-&gt;get('riotquest.reque...', Object(Closure))
#6 E:\XAMPP\htdocs\******\vendor\supergrecko\riot-quest\src\RiotQuest\Components\Engine\Request.php(105): RiotQuest\Components\Engine\Request-&gt;finalize()
#7 E:\XAMPP\htdocs\******\vendor\supergrecko\riot-quest\src\RiotQuest\Components\Endpoints\Match.php(61): RiotQuest\Components\Engine\Request-&gt;send()
#8 E:\XAMPP\htdocs\******\app\Http\Controllers\UserOrderController.php(93): RiotQuest\Components\Endpoints\Match-&gt;list('VLdpE_EHRBwNzP9...', Array)
#9 [internal function]: App\Http\Controllers\UserOrderController-&gt;index('624')
#10 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(54): call_user_func_array(Array, Array)
#11 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(45): Illuminate\Routing\Controller-&gt;callAction('index', Array)
#12 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Route.php(219): Illuminate\Routing\ControllerDispatcher-&gt;dispatch(Object(Illuminate\Routing\Route), Object(App\Http\Controllers\UserOrderController), 'index')
#13 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Route.php(176): Illuminate\Routing\Route-&gt;runController()
#14 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Router.php(680): Illuminate\Routing\Route-&gt;run()
#15 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): Illuminate\Routing\Router-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 E:\XAMPP\htdocs\******\app\Http\Middleware\isBuyer.php(25): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): App\Http\Middleware\isBuyer-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#18 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Middleware\SubstituteBindings.php(41): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Routing\Middleware\SubstituteBindings-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#21 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Auth\Middleware\Authenticate.php(43): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#23 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Auth\Middleware\Authenticate-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#24 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(75): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#27 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#29 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\View\Middleware\ShareErrorsFromSession-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#30 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(56): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Session\Middleware\StartSession-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#33 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#35 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#36 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(66): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#38 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Cookie\Middleware\EncryptCookies-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#39 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(104): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#41 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Router.php(682): Illuminate\Pipeline\Pipeline-&gt;then(Object(Closure))
#42 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Router.php(657): Illuminate\Routing\Router-&gt;runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#43 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Router.php(623): Illuminate\Routing\Router-&gt;runRoute(Object(Illuminate\Http\Request), Object(Illuminate\Routing\Route))
#44 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Router.php(612): Illuminate\Routing\Router-&gt;dispatchToRoute(Object(Illuminate\Http\Request))
#45 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(176): Illuminate\Routing\Router-&gt;dispatch(Object(Illuminate\Http\Request))
#46 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(30): Illuminate\Foundation\Http\Kernel-&gt;Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#47 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\TransformsRequest.php(21): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#48 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Foundation\Http\Middleware\TransformsRequest-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#49 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\TransformsRequest.php(21): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#51 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Foundation\Http\Middleware\TransformsRequest-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#52 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\ValidatePostSize.php(27): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#54 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Foundation\Http\Middleware\ValidatePostSize-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#55 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(62): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#57 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#58 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 E:\XAMPP\htdocs\******\vendor\fideloper\proxy\src\TrustProxies.php(57): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#60 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(163): Fideloper\Proxy\TrustProxies-&gt;handle(Object(Illuminate\Http\Request), Object(Closure))
#61 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(53): Illuminate\Pipeline\Pipeline-&gt;Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(104): Illuminate\Routing\Pipeline-&gt;Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#63 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(151): Illuminate\Pipeline\Pipeline-&gt;then(Object(Closure))
#64 E:\XAMPP\htdocs\******\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(116): Illuminate\Foundation\Http\Kernel-&gt;sendRequestThroughRouter(Object(Illuminate\Http\Request))
#65 E:\XAMPP\htdocs\******\public\index.php(55): Illuminate\Foundation\Http\Kernel-&gt;handle(Object(Illuminate\Http\Request))
#66 {main}
```

**Comment by supergrecko** [comment]
This is probably because the username has non-ascii characters in it and therefore the symfony cache denies it. I'll submit a patch to fix this.

**Comment by supergrecko** [comment]
Fixed in `1.3.6`
